diff --git a/node_modules/@sentry/nextjs/src/config/webpack.ts b/node_modules/@sentry/nextjs/src/config/webpack.ts
index 98bd56b..98292d0 100644
--- a/node_modules/@sentry/nextjs/src/config/webpack.ts
+++ b/node_modules/@sentry/nextjs/src/config/webpack.ts
@@ -66,18 +66,18 @@ export function constructWebpackConfigFunction(
     // the browser needs," but where it is, we can save ~1.6 kb in eventual bundle size by excluding code we know we
     // don't need. (Normally this would only matter for the client side, but because vercel turns backend code into
     // serverless functions, it's worthwhile to do it for both.)
-    if (buildContext.webpack.version.startsWith('5')) {
-      const excludedTracingDir = buildContext.isServer ? 'browser' : 'integrations/node';
-      newConfig.resolve = {
-        ...newConfig.resolve,
-        alias: {
-          ...newConfig.resolve?.alias,
-          [path.resolve(buildContext.dir, `./node_modules/@sentry/tracing/esm/${excludedTracingDir}`)]: false,
-          // TODO It's not clear if it will ever pull from `dist` (in testing it never does), so we may not need this.
-          [path.resolve(buildContext.dir, `./node_modules/@sentry/tracing/dist/${excludedTracingDir}`)]: false,
-        },
-      };
-    }
+    // if (buildContext.webpack.version.startsWith('5')) {
+    //   const excludedTracingDir = buildContext.isServer ? 'browser' : 'integrations/node';
+    //   newConfig.resolve = {
+    //     ...newConfig.resolve,
+    //     alias: {
+    //       ...newConfig.resolve?.alias,
+    //       [path.resolve(buildContext.dir, `./node_modules/@sentry/tracing/esm/${excludedTracingDir}`)]: false,
+    //       // TODO It's not clear if it will ever pull from `dist` (in testing it never does), so we may not need this.
+    //       [path.resolve(buildContext.dir, `./node_modules/@sentry/tracing/dist/${excludedTracingDir}`)]: false,
+    //     },
+    //   };
+    // }
 
     // Enable the Sentry plugin (which uploads source maps to Sentry when not in dev) by default
     const enableWebpackPlugin = buildContext.isServer
